flexmark-util/src/test/java/com/vladsch/flexmark/util/UtilsTest.java

```diff
@@ -1,14 +1,245 @@
 package com.vladsch.flexmark.util;
 
+import org.junit.Assert;
+import org.junit.Rule;
 import org.junit.Test;
+import org.junit.rules.ExpectedException;
 
-import static com.vladsch.flexmark.util.Utils.*;
-import static org.junit.Assert.*;
+import java.util.ArrayList;
 
 public class UtilsTest {
 
+    @Rule public ExpectedException thrown = ExpectedException.none();
```

Comment:
This & line 6 not necessary.

Reply:
We're expecting an exception at line 182